qep-320-backport-policy.md

```diff
@@ -0,0 +1,53 @@
+# QGIS Enhancement 320: Backport policies
+
+## Summary
+
+This QEP documents the standards and requirements for backporting changes from the ``master`` branch to
+Long Term Release (LTR) and current stable release branches in QGIS. Developers must follow these guidelines
+when submitting backports.
+
+Notes:
+
+- These guidelines describe QGIS specific backporting practices only. Following general
+  "best practice" for software maintenance and version control is assumed.
+- Different policies apply for backports to stable release branches versus LTR branches.
+
+## General Backporting Guidelines
+ 
+- 1.1. Changes must be merged to ``master`` before backporting.
+- 1.2. Backports must be submitted using Pull Requests.
+- 1.3. A backport must be approved by a core QGIS developer prior to merge.
+  - 1.3.1. The developer of the original change is not allowed to "self-approve"
+   their own backported changes.
+- 1.4. Changes which are permissible for backporting are:
+  - 1.4.1. Bug fixes
+  - 1.4.2. API modifications and additions which are suitably "low risk"
```

Comment:
If "low risk" needs quotes, then maybe it would be better to elaborate a bit. Maybe "which have a low risk of introducing a regression" or similar?

Reply:
Ok, fixed